mnist/mnist_vanilla_k8s.ipynb

```diff
@@ -0,0 +1,909 @@
+{
```

Comment:
<p>nit: I think its cleaner to use the python client libraries to get these values rather than shelling out to kubectl.</p><p> </p><p>More importantly is it possible to print a warning but keep going in the event the user lacks permission? </p><p> </p><p>Ideally, a user would just be able to click run all and it would run top to bottom. Since printing out the URLs isn't critical it would be best if it printed out a warning but otherwise kept going.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/examples/pull/737/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>Changed it out to use the k8s python client library and warn if user is not authenticated.</p><p>Added instructions on fetching the URLs at the end of the notebook in case the user wants to do it manually.</p>